softioc/imports.py

```diff
@@ -27,6 +27,17 @@ def install_pv_logging(acf_file):
     '''Install pv logging'''
     _extension.install_pv_logging(acf_file)
 
+def create_callback_capsule():
+    return _extension.create_callback_capsule()
+
+def signal_processing_complete(record, callback):
+    '''Signal that asynchronous record processing has completed'''
+    _extension.signal_processing_complete(
+        record.PRIO,
+        record.record.value,
+        callback
+    )
```

Comment:
Am sure this dangling `)` style is new to this code!

Reply:
Fixed in a0bce0ce330bb96274d4bbebff179c872165d7e6